custom/icds_reports/tests/outputs/agg_awc_2017-05-01_5_sorted.csv

```diff
@@ -1,56 +1,56 @@
-st2,d3,b4,s8,a8,2017-05-01,1,1,,,,,,,,,,,,,,,,,1,,0,7,,,,,,,,,,,1,2,12,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-01-14,pucca,,,,,1,1,1,,0,1,1,1,,,,no,,,,,,5,,,,,,80,2,2,,1,2,12,0,0,0,0,1,,0,,1,0,15,,,,0,,0,0,0,0,0,0,0,0,9,0,0,,3,3
-st3,d4,b5,s20,a101,2017-05-01,1,,,,,,,,,,,,,,,,,,,,0,1,,,,,,,,,,,0,1,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,yes,,,,,,5,,,,,1,,,,,0,1,1,,,,,,,,,0,0,2,,,,0,,0,0,0,0,1,0,0,0,1,0,0,,1,0
-st4,d5,b6,s21,a102,2017-05-01,1,,,,,,,,,,,,,,,,,,,,0,1,,,,,,,,,,,0,1,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,yes,,,,,,5,,,,,1,,,,,0,1,1,,,,,,,,,0,0,2,,,,0,,0,0,0,1,0,0,0,0,1,0,0,,1,0
-st5,d6,b7,s22,a103,2017-05-01,1,,,,,,,,,,,,,,,,,,,,0,1,,,,,,,,,,,0,1,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,yes,,,,,,5,,,,,1,,,,,0,1,1,,,,,,,,,0,0,2,,,,0,,0,0,1,0,0,0,0,0,1,0,0,,1,0
-st6,d7,b8,s23,a104,2017-05-01,1,,,,,,,,,,,,,,,,,,,,0,1,,,,,,,,,,,0,1,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,yes,,,,,,5,,,,,1,,,,,0,1,1,,,,,,,,,0,0,2,,,,0,,0,1,0,0,0,0,0,0,1,0,0,,1,0
-st7,d8,b9,s24,a105,2017-05-01,1,,,,,,,,,,,,,,,,,,,,0,1,,,,,,,,,,,0,1,1,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,yes,,,,,,5,,,,,1,,,,,0,1,1,,,,,,,,,0,0,2,,,,0,,1,0,0,0,0,0,0,0,1,0,0,,1,0
+st1,d1,b1,s1,a1,2017-05-01,1,18,,,,,,,,,,,,,,,,,1,,,,,,,,,,,,,,2,3,,2,0,0,0,0,0,0,0,0,0,0,0,0,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-05-13,semi_pucca,,,,,1,1,0,,0,0,0,1,0,0,0,no,,,,,,5,0,0,0,0,0,,1,1,,2,3,,0,0,0,0,0,,0,,1,1,5,,,,0,,0,0,0,0,0,0,0,0,12,,,,5,2
+st1,d1,b1,s1,a17,2017-05-01,1,11,,,,,,,,,,,,,,,,,1,,7,7,,,,,,,,,,,0,3,14,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,no,,,,,,5,0,0,0,0,0,,2,2,,0,3,14,0,0,0,0,,,0,,0,12,17,,,,0,,0,0,0,0,0,0,0,0,5,0,0,,3,1
+st1,d1,b1,s1,a25,2017-05-01,1,13,,,,,,,,,,,,,,,,,1,,15,15,,,,,,,,,,,4,4,23,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-05-26,,,,,,1,1,1,,0,0,0,1,,,,no,,,,,,5,0,0,0,0,0,,16,16,,4,4,23,1,0,1,0,1,,0,,1,21,31,,,,0,,0,0,0,0,0,0,0,0,11,0,0,,8,7
+st1,d1,b1,s1,a33,2017-05-01,1,12,,,,,,,,,,,,,,,,,1,,20,21,,,,,,,,,,,3,4,28,0,0,0,1,0,0,0,0,0,0,1,0,1,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-05-19,,,,,,1,1,1,,1,0,1,1,0,0,0,no,,,,,,5,0,0,0,0,0,,39,39,,3,4,28,2,1,2,1,1,,0,,1,22,35,,,,0,,0,0,0,0,0,0,0,0,13,1,1,,7,0
+st1,d1,b1,s1,a41,2017-05-01,1,16,,,,,,,,,,,,,,,,,1,,8,8,,,,,,,,,,,2,0,11,1,0,0,0,0,0,0,0,0,0,0,2,0,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-05-18,semi_pucca,,,,,1,0,1,,0,0,1,1,0,0,0,yes,,,,,,5,1,1,1,1,1,53,,,,2,0,11,,,,,1,,,,1,2,13,,,,0,,0,0,0,0,0,0,0,0,1,0,0,,2,0
+st1,d1,b1,s1,a49,2017-05-01,1,14,,,,,,,,,,,,,,,,,1,,17,17,,,,,,,,,,,3,5,19,0,0,0,1,0,0,0,0,0,1,0,0,0,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,0,0,0,yes,,,,,,5,1,1,1,1,1,145,16,16,,3,5,19,0,1,0,1,,,0,,0,14,27,,,,0,,0,0,0,0,0,0,1,1,23,4,4,,8,0
+st1,d1,b1,s1,a9,2017-05-01,1,18,,,,,,,,,,,,,,,,,1,,0,4,,,,,,,,,,,3,0,8,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,2017-01-12,,,,,,1,1,1,,0,1,1,1,,,,no,,,,,,5,0,0,0,0,0,,,,,3,0,8,,,,,1,,,,1,0,11,,,,0,,0,0,0,0,0,0,0,0,2,0,0,,3,0
```

Comment:
This could just be replaced 86 with 0 but because of this change https://github.com/dimagi/commcare-hq/pull/25354/files#diff-35c59d5d660c28397ba3f1e0b6562868R20. its all made 0 instead of just null to be have data consistent with itself.